resume.tex

```diff
@@ -0,0 +1,123 @@
+
+\input{command.tex}
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% Now for the actual document:
+
+\begin{document}
+
+\input{header.tex}
+
+%%%%%%%%%%%%%%%%%%%%%%%% 
+\begin{ressection}{Employment}
+	\begin{restitledposition}{Nurx / Thirty Madison}{Staff Software Engineer}{April 2022--August 2024}
+		\ressubitem{\textbf{Job Runner} Designed, implemented, and maintained an in-house asynchronous job running
+		solution. Using RabbitMQ, NestJS, and NodeJS/TypeScript, created a new standardized solution for running async
+		tasks at scale across a suite of microservices.}
+		\ressubitem{\textbf{Kafka Topic Isolation} Isolated Kafka topics within staging environments
+		leading to improved stability across dozens of staging environments leading to increased developer efficiency.}
+		\ressubitem{\textbf{Health Metric Standardization} Combining an internal NPM package, Terminus, and Kubernetes
+		probes developed a reliable set of metrics to monitor service health.}
+		\ressubitem{\textbf{Core Services} Supported a microservices architecture with tools and libraries including:
```

Comment:
can you add anything to these bullet points that highlights independence or self-organization? I'll always take an engineer who can self-direct over one I have to feed work to.

Reply:
`Supported a microservices architecture as part of a small independently led engineer team with tools and libraries including...`